DNN Platform/Tests/DotNetNuke.Tests.Web/InternalServices/FileUploadControllerTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿namespace DotNetNuke.Tests.Web.InternalServices
+{
+    using System.Data;
+    using System.Globalization;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Abstractions;
+    using Abstractions.Application;
+    using Abstractions.Logging;
+
+    using Common;
+    using Common.Lists;
+    using Common.Utilities;
+
+    using Data;
+    using DotNetNuke.Entities.Portals;
+    using DotNetNuke.Instrumentation;
+    using DotNetNuke.Web.InternalServices;
+
+    using Microsoft.Extensions.DependencyInjection;
+
+    using Moq;
+
+    using NUnit.Framework;
+
+    using Services.Cache;
+
+    using Utilities.Mocks;
+    /// <summary>
+    /// Tests FileUploadController methods
+    /// </summary>
+    [TestFixture]
+    public class FileUploadControllerTests
+    {
+        private Mock<ILoggerSource> _mockLoggerSource = new Mock<ILoggerSource>();
+        private Mock<DataProvider> _mockDataProvider;
+        private Mock<ICBO> _mockCBO = new Mock<ICBO>();
+        private FileUploadController _testInstance;
+        private Mock<CachingProvider> _mockCachingProvider;
+        private Mock<IPortalController> _mockPortalController;
+        private TestSynchronizationContext _synchronizationContext = new TestSynchronizationContext();
+
+        [SetUp]
+        public void SetUp()
+        {
+            LoggerSource.SetTestableInstance(this._mockLoggerSource.Object);
+            CBO.SetTestableInstance(this._mockCBO.Object);
+            this.SetupDataProvider();
+            this.SetupCachingProvider();
+            this.SetupPortalSettings();
+            this.SetupServiceProvider();
+            this.SetupSynchronizationContext();
+        }
+
+        [Test]
+        public async Task UploadFromLocal_ShouldUploadFile_WithTrCultureAsync()
+        {
+            Thread.CurrentThread.CurrentCulture = new CultureInfo("tr-TR");
```

Comment:
Consider using the `[SetCulture("tr-TR")]` attribute instead

Reply:
Update code to make use of Attribute.